docs/api/shell.md

```diff
@@ -53,9 +53,18 @@ example, mailto: URLs in the user's default mail agent).
 
 * `fullPath` String
 
+Returns `Promise` - Which is resolved to `undefined` in case of success or
```

Comment:
We just can't change the return type that easily. It will break the existing code in a non-trivial way:
```js
const removed = shell.moveItemToTrash(path);
if (!removed) {
    // Failure "handler" will always be executed now.
}
```

So I suggest to change the API in three steps:
1. Add `shell.moveItemToTrashSync()` and `shell.moveItemToTrashAsync()`, deprecate `shell.moveItemToTrash()` in favour of `shell.moveItemToTrashSync()` **NEW FEATURE, NON-BREAKING CHANGE**
2. Make `shell.moveItemToTrash()` to be async, deprecate `shell.moveItemToTrashAsync()` **BREAKING CHANGE**.
3. Remove `shell.moveItemToTrashAsync()` **BREAKING CHANGE**.